docs/UserGuide.md

```diff
@@ -183,26 +218,18 @@ This command is typically used to **check a guest in** to your hotel with their
 
 **Format**: `add n/NAME p/PHONE e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS [rq/REQUEST]`
 
-**Constraints of parameters**:
+<div markdown="span" class="alert alert-info">
 
-&nbsp;&nbsp;&nbsp;Unless stated otherwise, all fields **should not** be left blank.
-* `NAME` should only contain alphanumeric characters and spaces.
-* `PHONE` should only contain numbers, and it should be at least 3 digits and at most 15 digits long.
-* `EMAIL` should only contain alphanumeric characters and '@' character. The local segment (before "@") should be at most 64 characters long, and the domain segment (after "@") should be at most 255 characters long.
-* `ROOM` should only contain alphanumeric characters. A hyphen can only be added if it is in between 2 alphanumeric characters.
-* `DATE_RANGE` should follow the format dd/MM/yy - dd/MM/yy.
-* `NUMBER_OF_GUESTS` should only contain numbers, and it should be at least 1 and at most 4.
-* `REQUEST` is an optional field, and it should be at most 500 characters long.
+**:information_source: Additional information:**<br>
+* Refer to the [Constraints of fields](#constraints-of-fields) for more information about the limitations
+  of each field.
+</div>
 
 **Examples**:
-* `add n/John Doe p/98765432 e/johnd@example.com rm/05-73 dr/19/05/22 - 24/05/22 ng/3`
-* `add n/Betsy Crowe p/82297553 e/betsycrowe@example.com rm/Aloha5 dr/10/01/12 - 11/01/12 ng/1 rq/Extra towels`
+* `add n/John Doe p/98765432 e/johnd@example.com rm/05-73 dr/19/05/22 - 24/05/22 ng/3` Adds the guest _John Doe_, with his details, to GuestBook.
+* `add n/Betsy Crowe p/82297553 e/betsycrowe@example.com rm/Aloha5 dr/10/01/12 - 11/01/12 ng/1 rq/Extra towels` 
+Adds the guest _Betsy Crowe_ to GuestBook, with the `request` to have _extra towels in her room_.
 
-### Listing all guests: `list`
-
-This command displays a list of all guests in GuestBook.
-
-**Format**: `list`
 
```

Comment:
Should we add a line after every command to know when we're going to the next command? Like this.

![image](https://user-images.githubusercontent.com/87851444/199216904-659e6ee2-df59-4dbe-a190-0f871e42b1e4.png)

Reply:
Agreed!